src/rebar3_hank_prv.erl

```diff
@@ -115,11 +115,21 @@ get_rules(State) ->
     end.
 
 normalize(IgnoreRules) ->
-    lists:map(fun ({Wildcard, Rule, Options}) ->
-                      {Wildcard, Rule, Options};
-                  ({Wildcard, Rule}) ->
-                      {Wildcard, Rule, all};
-                  (Wildcard) ->
-                      {Wildcard, all, all}
+    lists:foldl(fun ({_, _, _} = WildcardRuleOpts, Acc) ->
+                        normalize_rules(WildcardRuleOpts, Acc);
+                    ({_, _} = WildcardRule, Acc) ->
+                        normalize_rules(WildcardRule, Acc);
+                    (Wildcard, Acc) ->
+                        [{Wildcard, all, all} | Acc]
               end,
+              [],
               IgnoreRules).
+
+normalize_rules({Wildcard, Rules, Options}, Acc) when is_list(Rules) ->
+    [{Wildcard, Rule, Options} || Rule <- Rules] ++ Acc;
+normalize_rules({Wildcard, Rule, Options}, Acc) ->
+    normalize_rules({Wildcard, [Rule], Options}, Acc);
```

Comment:
I could have just done

```erlang
[{Wildcard, Rule, Options} | Acc]
```

here, and below, as well, but I preferred to have the "handling" in a single (well, actually two single) clause(s).

Let me know if you prefer a different pattern here.

Reply:
I don't really mind… @pbrudnick anything to say?

Reply:
Makes sense to me 👍